src/services/about.rs

```diff
@@ -16,3 +18,18 @@ pub async fn about(context: &Context<'_>, chain_id: &str) -> ApiResult<About> {
         build_number: build_number(),
     })
 }
+
+pub async fn get_master_copies(
```

Comment:
Extracted for the sake of re-usability, but this internally created a new instance of the `info_provider` which in the case of the `safe/<address>` endpoint can have unforeseen consenquences.